### PR TITLE
[tool,web] Include more info URI on Wasm init failure

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

File: packages/flutter_tools/lib/src/web/web_constants.dart [hunk 1]
```diff
@@ -0,0 +1,5 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+const String kWasmMoreInfo = 'See https://flutter.dev/wasm for more information.';
```
Comment (Hixie) [inline packages/flutter_tools/lib/src/web/web_constants.dart:5]: i think we already have a file with constant strings, fwiw.

File: packages/flutter_tools/lib/src/web/file_generators/wasm_bootstrap.dart [hunk 2]
```diff
@@ -18,6 +20,7 @@ String generateWasmBootstrapFile(bool isSkwasm) {
     moduleInstance = await dart2wasm_runtime.instantiate(dartModulePromise, imports);
   } catch (exception) {
     console.error(`Failed to fetch and instantiate wasm module: \${exception}`);
+    console.error($kWasmMoreInfo);
```
Comment (Hixie) [inline packages/flutter_tools/lib/src/web/file_generators/wasm_bootstrap.dart:23]: you should have a test for the bug here, at least. :-)

Comment (kevmoo) [reply packages/flutter_tools/lib/src/web/file_generators/wasm_bootstrap.dart:23]: The bug is running a browser without the right flags. I can add a TODO for @eyebrowsoffire

File: packages/flutter_tools/lib/src/web/web_constants.dart (hunk 1 above)
Comment (kevmoo) [reply packages/flutter_tools/lib/src/web/web_constants.dart:5]: We have `UserMessages` but there's talk of removing it - https://github.com/flutter/flutter/issues/125155

I don't see any other good candidates (I did look...)

File: packages/flutter_tools/lib/src/web/file_generators/wasm_bootstrap.dart (hunk 2 above)
Comment (kevmoo) [reply packages/flutter_tools/lib/src/web/file_generators/wasm_bootstrap.dart:23]: Oh 🤦

**Comment by kevmoo** [comment]
@Hixie – could I get an exemption? 😄

**Review by mdebbar — APPROVED** [review]

**Comment by Hixie** [comment]
It looks like this was landed without tests and without a test exemption.

**Comment by kevmoo** [comment]
> It looks like this was landed without tests and without a test exemption.

Gah! I saw your comments...like "you looked it it", but you're right. Uh. How would you like me to proceed?

**Comment by Hixie** [comment]
> > It looks like this was landed without tests and without a test exemption.
> 
> Gah! I saw your comments...like "you looked it it", but you're right. Uh. How would you like me to proceed?

Well ideally we'd have a test that would catch the error that was in the original version of the PR -- I'm not sure if there's a JS linter or something we could apply to the output we generate? But that would be ideal...

**Pull request merged**